src/transformers/models/layoutlm/modeling_layoutlm.py

```diff
@@ -1285,7 +1285,9 @@ def forward(
         >>> import torch
 
         >>> tokenizer = AutoTokenizer.from_pretrained("impira/layoutlm-document-qa", add_prefix_space=True)
-        >>> model = LayoutLMForQuestionAnswering.from_pretrained("impira/layoutlm-document-qa")
+        >>> model = LayoutLMForQuestionAnswering.from_pretrained(
+        ...     "impira/layoutlm-document-qa", revision="1e3ebacae311132a7c4a8df2d350d7e4ffd6ff0a"
```

Comment:
Note that you only need to put 7 characters for the commit hash, might be less intimating (and fit in one line) this way.

Reply:
Good point. Just updated.